sequel/where.js

```diff
@@ -95,7 +95,7 @@ WhereBuilder.prototype.single = function single(queryObject, options) {
     var population = queryObject.instructions[attr].instructions[0];
     var alias = utils.escapeName(utils.populationAlias(population.alias), self.escapeCharacter);
 
-    var parentAlias = _.find(_.values(self.schema), {tableName: population.parent}).identity;
+    var parentAlias = _.find(_.values(self.schema), {tableName: population.parent}).tableName;
```

Comment:
This seems right, based upon how it's being used, but the variable name has become confusing.  It's no longer an alias for the parent table.